<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFileDialog</class>
 <widget class="QDialog" name="SendFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送文件对话框</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cc-client.qrc">
    <normaloff>:/Icons/sendFile.png</normaloff>:/Icons/sendFile.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;选择要发送的文件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="sendFileNamesPlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sendDirCheckBox">
       <property name="text">
        <string>发送文件夹</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onlySendChangedFileCheckBox">
       <property name="toolTip">
        <string>是否只发送上次发送后修改过的文件</string>
       </property>
       <property name="text">
        <string>只发送修改过的文件</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browserPushButton">
       <property name="text">
        <string>选择文件...</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/search.ico</normaloff>:/Icons/search.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>目标文件位置：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destLineEdit">
       <property name="text">
        <string>/home</string>
       </property>
       <property name="placeholderText">
        <string>输入文件在工作站上的保存位置，不能为空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectDestPushButton">
       <property name="text">
        <string>选择文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>开始发送</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/52design.com_jingying_059.png</normaloff>:/Icons/52design.com_jingying_059.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>712</width>
        <height>384</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cc-client.qrc"/>
 </resources>
 <connections/>
</ui>
